eblob-bf.spec

```diff
@@ -64,6 +64,18 @@ appending records one after another.
 This package contains libraries, header files and developer documentation
 needed for developing software which uses the eblob library.
 
+%package python
```

Comment:
You are likely the first person ever who does this. Anyway, what is the purpose of having python in a separate package? Not C++, not devel, but python ony?